spec/terminology.md

```diff
@@ -3,17 +3,74 @@
 ::: todo
 :::
 
-[[def: method-specific identifier, MSI]]
-~ The `method-specific-id` part of DID Syntax, as defined in [DID Core](https://www.w3.org/TR/did-core/#did-syntax). See section [Method-Specific Identifier](#method-specific-identifier).
+[[def: authentic chained data container (ACDC), ACDC]]
+~ a variant of [the Verifiable Credential (VC) specification](https://www.w3.org/TR/vc-data-model/). Some of the major distinguishing features of ACDCs include normative support for chaining, use of composable JSON Schema, multiple serialization formats, support for Ricardian contracts, support for chain-link confidentiality compact formats for resource constrained applications, simple partial disclosure mechanisms and simple selective disclosure mechanisms. ACDCs inherit the security model derived from [[ref: KERI]].
+
+[[def: autonomic identifier (AID)]]
+~ A self-managing identifier that is self-certifying (i.e., self-authenticating) and cryptographically bound to a [[ref: key event log]] ([[ref: KEL]]). An AID is either non-transferable or transferable. A non-transferable AID does not support key rotation while a transferable AID support key rotation using a key [[ref: pre-rotation]] mechanism that enables the AID to persist in spite of the evolution of its key state.
+
+[[def: compact Event Streaming Representation (CESR), CESR]]
+~ A dual text-binary encoding format that has the unique property of text-binary concatenation composability. The composability property enables the round trip conversion en-masse of concatenated primitives between the text domain and binary domain while maintaining the separability of individual primitives.
```

Comment:
Could we do something that is more understandable?  Perhaps:

The text-binary data format in which the KERI events for the DID are stored. The benefit of the CESR format for the `did:webs` DID Method is the KERI events are additive (appended to the end of the file), which allows a resolver that has cached the KERI event data (state and events) to continue processing following their last processed event versus always starting from the beginning of the log. See the [CESR Specification](…) for more details about the format.

Reply:
How about "An encoding format that enables round-trip text-binary conversion of concatenated cryptographic primitives and general data types"?

- CESR is not just for KERI key events. The essence of CESR is to allow cryptographic primitives to be concatenated with general data types such that they can be converted back and forth between binary and text.
- I think the property that "a resolver can cache key event data" is not unique to CESR.